src/apps/console/routes/_main+/$account+/packages+/access-management/cred-resources.tsx

```diff
@@ -1,8 +1,5 @@
-import { CaretDownFill, Copy, Eye, Spinner, Trash } from '@jengaicons/react';
+import { Copy, Spinner, Trash, Check } from '@jengaicons/react';
```

Comment:
Icons should not be imported directly. have to import thorough console/components/icons.